lib/components/viewers/route-details.tsx

```diff
@@ -85,27 +86,29 @@ class RouteDetails extends Component<Props> {
         ([id, pat]): PatternSummary => ({
           geometryLength: pat.patternGeometry?.length || 0,
           headsign: extractHeadsignFromPattern(pat, shortName),
-          id
+          id,
+          lastStop: pat.stops?.[pat.stops?.length - 1]?.name
         })
       )
-      // Remove duplicate headsigns. Using a reducer means that the first pattern
-      // with a specific headsign is the accepted one. TODO: is this good behavior?
+      // Remove duplicate headsigns. Replaces duplicate headsigns with the last stop name
```

Comment:
Nitpicky but we're not technically removing anything anymore right, just appending the last stop